src/models/mockFactory.ts

```diff
@@ -0,0 +1,200 @@
+import ProjectSettingsPage,
+{ IProjectSettingsPageProps } from "../react/components/pages/projectSettings/projectSettingsPage";
+import IProjectActions, * as projectActions from "../redux/actions/projectActions";
+import { IProject, IConnection, IExportFormat,
+    ITag, IAsset, IApplicationState, IAppSettings,
+    IAssetMetadata, ISize } from "../models/applicationState";
+import { IProjectService } from "../services/projectService";
+import IConnectionActions, { loadConnection, deleteConnection } from "../redux/actions/connectionActions";
+
+export class MockFactory {
+    public tags(): ITag[] {
+        return [
+            {
+                name: "Tag1",
+                color: "#FFFFFF",
+            },
+            {
+                name: "Tag2",
+                color: "#FFFF00",
+            },
+        ];
+    }
+
+    public connection(): IConnection {
+        return {
+            id: "connectionId",
+            name: "My new connection",
+            description: "This is my description",
+            providerType: "localFileSystemProxy",
+            providerOptions: {
+                folderPath: "my path",
+            },
+        };
+    }
+
+    public connections(): IConnection[] {
+        return [
+            {
+                id: "1",
+                name: "My source connection",
+                description: "This is my connection",
+                providerType: "azureBlobStorage",
+                providerOptions: {
+                    connectionString: "myconnectionstring",
+                    containerName: "container",
+                    createContainer: true,
+                },
+            },
+            {
+                id: "2",
+                name: "My target connection",
+                description: "This is my connection",
+                providerType: "localFileSystemProxy",
+                providerOptions: {
+                    folderPath: "my path",
+                },
+            },
+        ];
+    }
+
+    public exportFormat(): IExportFormat {
+        return {
+            providerType: "Fake",
+            providerOptions: {},
+        };
+    }
+
+    public project(): IProject {
```

Comment:
There is some overlap here vs. what i recently introduced - Let's consolidate

Reply:
Will do. I'll make them all static and put them in your class